buildscripts/pre_release.sh

```diff
@@ -45,7 +45,7 @@ if [ ! -f $changelog_file ]; then
     exit 1
 fi
 
-if [ ! grep -q "## [Unreleased]" $changelog_file ]; then
+if [ ! grep -q "\#\# \[Unreleased\]" $changelog_file ]; then
```

Comment:
nit: add `^` to be beginning of the patter to match from the beginning of the line?

Reply:
Fixed -added `^` in beginning and `$` in end to search for whole word.